### ISSUE TITLE
Integrate highlight-loader and highlight.js

### ISSUE THREAD

**Issue by bebraw** [antwarjs/antwarjs.github.io#2]
See https://www.npmjs.com/package/highlight-loader . It would take a bit more Webpack config. Maybe sites themselves should be able to tweak the basic conf?

**Comment by eldh** [comment]
Perhaps. You could also do highlight integration in the postProcess step, right? What's the benefit of doing this in a loader? Atm, the markdown conversion is done after the file has been loaded (because we first need to seperate the front-matter).

**Comment by bebraw** [comment]
Yeah, this is something that needs to get applied on HTML. I just realized `yaml-headmatter-loader` outputs JSON so there isn't a nice way to pipe here as far as I can see.

The right place to do this would be just after the Markdown conversion step.

**Comment by eldh** [comment]
Either override `content` or use `postProcessPosts`, I'd go for the latter.
